os/provisioning.md

```diff
@@ -30,25 +32,27 @@ etcd:
   discovery:                   https://discovery.etcd.io/<token>
 ```
 
-But what are `PUBLIC_IPV4` and `PRIVATE_IPV4`? These variables are populated from the environment in which Container Linux runs (e.g. EC2, Azure, GCE), assuming this metadata exists. In the default EC2 case, the `public_ipv4` and `local_ipv4` metadata would be used. On bare metal, this information cannot be reliably derived in a general manner, so these variables couldn't be used. In Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` and the dynamic IP would be used for the `PRIVATE_IPV4`. These are just a few examples. Given the many different environments in which Container Linux can run, it's difficult if not impossible to accurately determine these variables in every scenario.
+`PUBLIC_IPV4` and `PRIVATE_IPV4` are automatically populated from the environment (such as EC2, Azure, or GCE) in which Container Linux runs, if this metadata exists. For example: For EC2 (the default), the `public_ipv4` and `local_ipv4` metadata would be used. In Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` and the dynamic IP would be used for the `PRIVATE_IPV4`. On bare metal, this information cannot be reliably derived in a general manner, so these variables cannot be used.
```

Comment:
I see. so, questions: This doc should walk users through the standard implementation, which it does for most of the doc. 1) Are all three of these platforms edge cases? and, more importantly 2) when you say 'are populated' what you mean is 'Populate these variables according to the environment in which CL runs.' Correct? I wasn't paying enough attention to the code, apparently. I saw curly brackets, and didn't think variable. 3) Also, any reason not to use one standard for variables - either {} or <> ?